src/main/resources/db/migration/V202409081445__create_owner_table.sql

```diff
@@ -0,0 +1,21 @@
+create table owner
```

Comment:
same here

Reply:
I would appreciate an explanation, I wrote the date as the name of the file 202409081445 at the beginning.

Reply:
you are correct, my mistake ✌️